tests/test_ipyvizzu.py

```diff
@@ -241,14 +224,17 @@ def test_store(self):
             Config({"color": {"set": ["Genres"]}}), Style({"legend": {"width": 50}})
         )
         snapshot_b = self.chart.store()
-        self.chart.animate(snapshot_a)
+        self.chart.animate(snapshot_a, duration="4s")
         self.chart.animate(snapshot_b)
-        self.chart.show()
         self._assert_display("store.html")
 
     def _assert_display(self, asset_name):
         asset_path = self.asset_dir / asset_name
+        display_out = ""
+        for block in self.display_html.call_args_list:
+            for line in block.args[0].split("\n"):
+                display_out += line.strip() + "\n"
```

Comment:
This is not a big difference here but it is better to avoid `+=` for `str`. Collecting `str` into list and `join` them is more preferable.

Reply:
thanks, I resolved it in #49